<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="1,0,0">
   <item row="4" column="1">
    <widget class="QSpinBox" name="maxOutputLineSB">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="specialValueText">
      <string>Unlimited</string>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>400</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="clearWorkingDirsPB">
     <property name="toolTip">
      <string>Clear the working directory history</string>
     </property>
     <property name="statusTip">
      <string>Clear the working directory history</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="mmviiWindowCB">
     <property name="toolTip">
      <string>Open a window to display MMVII execution results</string>
     </property>
     <property name="statusTip">
      <string>Open a window to display MMVII execution results</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Maximum number of lines displayed in MMVII ouput window</string>
     </property>
     <property name="statusTip">
      <string>Maximum number of lines displayed in MMVII ouput window</string>
     </property>
     <property name="text">
      <string>Max &amp;lines in output :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>maxOutputLineSB</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>Open a window to display MMVII execution results</string>
     </property>
     <property name="statusTip">
      <string>Open a window to display MMVII execution results</string>
     </property>
     <property name="text">
      <string>MMVII &amp;output window :</string>
     </property>
     <property name="buddy">
      <cstring>mmviiWindowCB</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>Maximum number of previous commands displayed in the history list</string>
     </property>
     <property name="statusTip">
      <string>Maximum number of previous commands displayed in the history list</string>
     </property>
     <property name="text">
      <string>Max command &amp;history :</string>
     </property>
     <property name="buddy">
      <cstring>maxCommandHistorySB</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Clear the working directory history</string>
     </property>
     <property name="statusTip">
      <string>Clear the working directory history</string>
     </property>
     <property name="text">
      <string>&amp;Working dir list :</string>
     </property>
     <property name="buddy">
      <cstring>clearWorkingDirsPB</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="maxCommandHistorySB">
     <property name="toolTip">
      <string>Maximum number of previous commands displayed in the history list</string>
     </property>
     <property name="statusTip">
      <string>Maximum number of previous commands displayed in the history list</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Police size in output :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>outputFontSizeSB</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="outputFontSizeSB">
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>18</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clearWorkingDirsPB</tabstop>
  <tabstop>maxCommandHistorySB</tabstop>
  <tabstop>mmviiWindowCB</tabstop>
  <tabstop>maxOutputLineSB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
